manifests/icp_access.pp

```diff
@@ -16,15 +16,13 @@
 #   Order can be used to configure where in `squid.conf`this configuration section should occur.
 define squid::icp_access (
   Enum['allow', 'deny']
-          $action = 'allow',
+  $action = 'allow',
```

Comment:
I guess they wanted alignment here. It's why I think alignment is bad. IMHO the type and variable should be on the same line here.

Reply:
I agree. I just think the autofix wasn't perfect here. I will fix it